.eslintrc.yml

```diff
@@ -25,18 +27,325 @@ rules:
+  # require-await >> Asynchronous closures disabled by no-restricted-syntax
   require-unicode-regexp: error
+  # require-yield >> Generator closures disabled by no-restrict-syntax
+  # rest-spread-spacing: [error, never] >> Prettier
+  # semi-spacing: [error, {before:false, after:false}] >> Prettier
+  # semi-style: [error, last] >> Prettier
+  semi: [error, always]
+  # sort-imports
+  # sort-keys
+  # sort-vars
+  # space-before-blocks: [error, always] >> Prettier
+  # space-before-function-paren: [error, always] >> Prettier
+  # space-in-parens: [error, never] >> Prettier
+  # space-infix-ops: [error] >> Prettier
+  # space-unary-ops: [error, {words:true, nonwords:false}] >> syle
+  # spaced-comment: [error, always] >> Prettier
+  strict: [error, never]
+  # switch-colon-spacing >> cf no-restricted-syntax >> SwitchStatement
+  # symbol-description >> No Symbol global
+  template-curly-spacing: [error, never]
+  # template-tag-spacing: [error, never] >> Prettier
+  unicode-bom: [error, never]
+  # use-isnan >> Heuristic
+  # valid-jsdoc >> Deprecated
+  valid-typeof: [error, {requireStringLiterals: true}]
+  # vars-on-top >> Variables are disabled by no-restricted-syntax
+  # wrap-iife: [error, outside] >> Function disabled by no-restricted-syntax
+  # wrap-regex: error
+  # yield-star-spacing >> Generator closures disabled by no-restrict-syntax
+  yoda: [error, never]
+
+  #################
+  # Plugin Import #
+  #################
+  import/default: error
+  # import/dynamic-import-chunkname: error
+  import/export: error
+  # import/exports-last: error
+  # import/extensions: error
   import/first: error
+  # import/group-exports: error
+  # import/max-dependencies: error
+  import/named: error
+  # import/namespace
+  # import/newline-after-import: error
+  import/no-absolute-path: error
+  # import/no-amd: error
+  # import/no-anonymous-default-export: error
+  # import/no-commonjs: error
+  import/no-cycle: error
+  # import/no-default-export: error
+  import/no-deprecated: error
+  import/no-duplicates: error
+  # import/no-dynamic-require: error
   import/no-extraneous-dependencies: error
+  # import/no-import-module-exports: error
+  # import/no-internal-modules: error
+  import/no-mutable-exports: error
+  # import/no-named-as-default-member: error
+  # import/no-named-as-default: error
+  import/no-named-default: error
+  # import/no-named-export: error
+  # import/no-namespace: error
+  # import/no-nodejs-modules: error
+  # import/no-relative-packages: error
+  # import/no-relative-parent-imports: error
+  # import/no-restricted-paths
+  import/no-self-import: error
+  # import/no-unassigned-import: error
   import/no-unresolved: error
+  # import/no-unused-modules: error
   import/no-useless-path-segments: error
+  # import/no-webpack-loader-syntax: error
+  # import/order: error
+  # import/prefer-default-export: error
+  # import/unambiguous: error
```

Comment:
Yeah I went from `airbnb` to `eslint:recommended` to none.

Reply:
You want me to remove the commented rules?

Reply:
Not necessarily, it makes sense to leave them if there's no preset being used. I just wonder why not use a preset. At the very least, it can make taking advantage of any new rules that might get implemented in the future easier.

Reply:
So there is `preset + enabling rules + disabling rules` versus `enabling rules` only. Even if the former is smaller in size, it is less straightforward and you don't know exactly what you are linting. Because there are actually not that many rules and that it is is easy to maintain, I prefer the later.